### DISCUSSION TITLE
Confused with getSession - I see other users information instead of mine

### DISCUSSION THREAD

**Discussion by toiluj23** [nextauthjs/next-auth#1047] [Help] [answered]
Hi guys, I have a doubt, and perhaps you can help me here:
When I'm inside **[real-domain.com]/api/auth/session** I see the last user it's logged in, instead of mine. (This issue is happening to me when I use a third party Provider, like Facebook or Google).

Do you know if that's something miscoded from my side (following the nextAuth doc) or if it's something related to the Backend?

Hope you can help me and share your experience about it. 

Have a wonderful beginning of 2021!

**Comment by iaincollins** [comment, marked as answer]
Hi there!

Can you share what database adapter you are using? Knowing any NextAuth.js configuration option you have set would also be helpful. If you are able to share a project repo, that would be ideal.

Thanks!

**Reply by toiluj23** [reply]
Iain, thanks for your response, Now it's working OK sorry. I figured out that when it's SSR if i do not use userData = null it shows me the latest user logged in every session.

```
jwt: async (token, user, account, profile, isNewUser) => {
            //console.log("[jwt]", userData)
            if(userData){
                user && (token.user = userData);
                **userData = null;**
            }
            else {
                user && (token.user = user);
            }
            return Promise.resolve(token)

        }
```

**Reply by iaincollins** [reply]
Oh! Interesting, thanks for the follow up.

I guess the process is sticking around and it's just using whatever value it has in memory from the last time it ran?
